packages/plugins/plugin-meeting/src/types/schema.ts

```diff
@@ -4,25 +4,40 @@
 
 import { Schema as S } from 'effect';
 
-import { Ref, TypedObject } from '@dxos/echo-schema';
-import { TreeType } from '@dxos/plugin-outliner/types';
-import { ChannelType } from '@dxos/plugin-space/types';
-import { TranscriptType } from '@dxos/plugin-transcription/types';
-import { TextType } from '@dxos/schema';
+import { Expando, Ref, TypedObject } from '@dxos/echo-schema';
 
 // TODO(wittjosiah): Factor out. Brand.
 const IdentityDidSchema = S.String;
 
 export const MeetingSchema = S.Struct({
+  /**
+   * User-defined name of the meeting.
+   */
   name: S.optional(S.String),
+  /**
+   * The time the meeting was created.
+   * Used to generate a fallback name if one is not provided.
+   */
+  created: S.String.annotations({ description: 'ISO timestamp' }),
+  /**
+   * List of dids of identities which joined some portion of the meeting.
+   */
   participants: S.mutable(S.Array(IdentityDidSchema)),
-  channel: S.optional(Ref(ChannelType)),
-  transcript: S.optional(Ref(TranscriptType)),
-  notes: S.optional(Ref(TreeType)),
-  summary: S.optional(Ref(TextType)),
+  /**
+   * Set of artifacts created during the meeting.
+   * Keys are the typename of the artifact.
+   * Values are a reference to the artifact object.
+   * For example, a meeting may have a transcript, notes, and a summary.
+   */
+  artifacts: S.mutable(
+    S.Record({
+      key: S.String,
+      value: Ref(Expando),
+    }),
+  ),
 });
 
 export class MeetingType extends TypedObject({
   typename: 'dxos.org/type/Meeting',
-  version: '0.1.0',
+  version: '0.2.0',
```

Comment:
while plugins are in experimental do we need to do major bumps?

Reply:
we have existing meeting objects in our production spaces because we're using them on labs now, so it's necessary. I don't think we need to be shy about version bumps when they're necessary